.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,105 @@
+name: Python Benchmark
+
+on:
+  push:
+  pull_request:
+
+concurrency:
+  group: tests-${{ github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    working-directory: ./bindings/python
+    shell: bash
+
+jobs:
+
+  pre-commit:
```

Comment:
Pre-commit is duplicated from tests-python.yml and not needed here.

Reply:
Done.